### PR TITLE
Refine documents of PARL

### PR DESCRIPTION
**Pull request by zenghsh3**
- remove not used files
- add benchmark for DQN and DDPG
- add Parameters management Readme
- fix dependence of examples

### CODE REVIEW

File: parl/framework/model_base.py [hunk 1]
```diff
@@ -124,23 +124,39 @@ class Model(Network):
     In conclusion, Model is responsible for forward and 
     Algorithm is responsible for backward.
 
-    Model can also be used to construct target model, which has the same structure as initial model.
+    Model can also use deepcopy way to construct target model, which has the same structure as initial model. 
+    Note that parameters in the initial model havn't been copied to target model.
```
Comment (TomorrowIsAnOtherDay) [inline parl/framework/model_base.py]: Note that only the model definition are copied here. To copy the parameters from the current model to the target model, you must explicitly use `sync_params_to`  function after the program is initialized.

File: parl/layers/layer_wrappers.py [hunk 2]
```diff
@@ -13,6 +13,21 @@
 # limitations under the License.
 """
 Wrappers for fluid.layers so that the layers can share parameters conveniently.
```
Comment (TomorrowIsAnOtherDay) [inline parl/layers/layer_wrappers.py]: Wrappers for fluid.layers. It helps to easily share parameters between layers.

**Review by TomorrowIsAnOtherDay — CHANGES_REQUESTED** [review]

File: parl/framework/model_base.py (hunk 1 above)
Comment (zenghsh3) [reply parl/framework/model_base.py]: done

File: parl/layers/layer_wrappers.py (hunk 2 above)
Comment (zenghsh3) [reply parl/layers/layer_wrappers.py]: done

**Review by TomorrowIsAnOtherDay — APPROVED** [review]

**Pull request merged**